zerver/views/auth.py

```diff
@@ -497,6 +500,20 @@ def log_into_subdomain(request: HttpRequest, token: str) -> HttpResponse:
                                          multiuse_object_key=multiuse_object_key,
                                          full_name_validated=full_name_validated)
 
+def store_login_data(data: Dict[str, Any]) -> str:
+    key = put_dict_in_redis(redis_client, LOGIN_KEY_FORMAT, data,
+                            expiration_seconds=LOGIN_KEY_EXPIRATION_SECONDS,
+                            token_length=LOGIN_TOKEN_LENGTH)
+    token = key.split(LOGIN_KEY_PREFIX)[1]  # remove the prefix
```

Comment:
This is probably broken if the key contains a second copy of LOGIN_KEY_PREFIX (very unlikely).  We should use `.split(LOGIN_KEY_PREFIX, 1)` to address that.

Reply:
Changed, in the ``auth: Use tokens, with data stored in redis, for log_into_subdomain.`` commit.